doc/tutorials/lattice_boltz_analysis.rst

```diff
@@ -195,6 +203,10 @@ the `Experiment` object, the following can be called to stop any database instan
     # be sure to be in Python environment where SmartSim is installed
     $(smart --dbcli) -h 127.0.0.1 -p 6780 shutdown
```

Comment:
this is just ``smart dbcli`` now

Reply:
Looks like duplicate of last comment, resolving